release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -3103,3 +3103,34 @@ $ oc adm release info 4.13.9 --pullspecs
 ==== Updating
 
 To update an existing {product-title} 4.13 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI] for instructions.
+
+[id="ocp-4-13-10"]
+=== RHSA-2023:4731 - {product-title} 4.13.10 bug fix and security update
+
+Issued: 2023-08-30
+
+{product-title} release 4.13.10, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2023:4731[RHSA-2023:4731] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2023:4734[RHBA-2023:4734] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.13.10 --pullspecs
+----
+
+[id="ocp-4-13-10-bug-fixes"]
+==== Bug fixes
+* Previously, clusters using Mint mode and had their root secret removed would encounter an issue during an upgrade from 4.13.8 to 4.13.9. This was caused by a modification in the credentials request of the Ingress Operator that was backported to 4.13.9. With this update, these clusters have no issues updating to versions 4.13.9 and greater. (link:https://issues.redhat.com/browse/OCPBUGS-17733[*OCPBUGS-17733*])
+
+[id="ocp-4-13-10-known-issue"]
+==== Known issue
+* The addition of a new feature in {product-title} 4.12 that enables UDP generic receive offload (GRO) GRO also causes all virtual ethernet pair (veth) devices to have one RX queue per available CPU. Previously each veth had one queue. Those queues are dynamically configured by Open Virtual Network (OVN) and there is no synchronization between latency tuning and this queue creation.
++
+The latency tuning logic monitors the veth NIC creation events and starts configuring the Receive Packet Steering (RPS) queue CPU masks before all the queues are properly created. This means that some of the RPS queue masks are not configured. Since not all NIC queues are configured properly, there is a chance of latency spikes in a real time application that uses timing sensitive CPUs for communicating with services in other containers. Applications that do not use kernel networking stack are not affected. (link:https://issues.redhat.com/browse/OCPBUGS-17794[*OCPBUGS-17794*])
```

Comment:
```suggestion
The latency tuning logic monitors the veth NIC creation events and starts configuring the Receive Packet Steering (RPS) queue CPU masks before all the queues are properly created. This means that some of the RPS queue masks are not configured. Since not all NIC queues are configured properly, there is a chance of latency spikes in a real-time application that uses timing sensitive CPUs for communicating with services in other containers. Applications that do not use kernel networking stack are not affected. (link:https://issues.redhat.com/browse/OCPBUGS-17794[*OCPBUGS-17794*])
```